fmriprep/config.py

```diff
@@ -122,8 +122,11 @@
     del _cgroup
 
 _fs_license = os.getenv('FS_LICENSE')
-if _fs_license is None and os.getenv('FREESURFER_HOME'):
-    _fs_license = os.path.join(os.getenv('FREESURFER_HOME'), 'license.txt')
+if not _fs_license and os.getenv('FREESURFER_HOME'):
+    _fs_home = os.getenv('FREESURFER_HOME')
+    if _fs_home and (Path(_fs_home) / "license.txt").exists():
```

Comment:
Yup, and it will correctly find the license bound in there:

https://github.com/poldracklab/fmriprep/blob/eb3a40b827d277ce661e31c27f6ace40ef3c2be6/wrapper/fmriprep_docker.py#L401-L404

Reply:
(actually, it shouldn't point to a directory, as I understand this)

Reply:
We check already:

- Using command line:
https://github.com/poldracklab/fmriprep/blob/eb3a40b827d277ce661e31c27f6ace40ef3c2be6/fmriprep/cli/parser.py#L418

- Using `fmriprep-docker`: because the container defines `FREESURFER_HOME=/opt/freesurfer` and the wrapper mounts in `$FREESURFER_HOME/license.txt`, then the config will check that condition at the line you commented on.